web-ng/src/app/components/form-field-editor/form-field-editor.component.ts

```diff
@@ -106,16 +111,16 @@ export class FormFieldEditorComponent implements OnInit, OnChanges, OnDestroy {
     });
   }
 
-  private validateLabel() {
-    if (
-      this.label ||
-      (!this.label &&
-        this.fieldCount === 1 &&
-        this.fieldType === FieldType.TEXT)
-    ) {
-      return null;
-    }
-    return { labelInvalid: true };
+  private validateLabel(control: FormControl) {
+    return this.isFormEmpty() ? null : Validators.required(control);
+  }
+
+  private isFormEmpty(): boolean {
+    return (
```

Comment:
"()" are not necessary here, please remove.

Reply:
Unfortunately I tried this but this will throw error on npm run check and I didn't do this intentionally it is done by prettier.

Reply:
Interesting, I've never seen that before. Thanks for letting me know, I'll look into it!